### PR TITLE
[Components with more than 1 service] LPS-177193, LPS-177207

### PR DESCRIPTION
**Pull request by daiazius**
Related issues: https://issues.liferay.com/browse/LPS-177193, https://issues.liferay.com/browse/LPS-177207

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by rafaprax** [comment]
@daiazius  Let's apply the same pattern here https://github.com/liferay-core-infra/liferay-portal/pull/3866

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
Pushed with no changes in the files, I've only changed the commit pattern

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/PinDTOConverter.java [hunk 1]
```diff
@@ -78,7 +79,7 @@ public Pin toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 	@Reference
 	private CSDiagramPinService _csDiagramPinService;
 
-	@Reference
+	@Reference(target = DTOConverterConstants.MAPPED_PRODUCT_DTO_CONVERTER)
```
Comment (rafaprax) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/PinDTOConverter.java:82]: Hey @daiazius, you forgot to change to DTOConverter here. please fix it.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/SpecificationDTOConverter.java [hunk 2]
```diff
@@ -78,7 +79,7 @@ public Specification toDTO(DTOConverterContext dtoConverterContext)
 	@Reference
 	private CPSpecificationOptionService _cpSpecificationOptionService;
 
-	@Reference
+	@Reference(target = DTOConverterConstants.OPTION_CATEGORY_DTO_CONVERTER)
```
Comment (rafaprax) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/SpecificationDTOConverter.java:82]: Hey @daiazius, you forgot to change to DTOConverter here. please fix it.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Hey @tinatian, this pr is ready for your review. Thx

**Pull request closed without merging**